.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,67 @@
+---
+name: Dependency review
+
+on:
+  merge_group:
+    types:
+      - checks_requested
+  pull_request:
+
+# Set a default shell for any run steps. The `-Eueo pipefail` sets errtrace,
+# nounset, errexit, and pipefail. The `-x` will print all commands as they are
+# run. Please see the GitHub Actions documentation for more information:
+# https://docs.github.com/en/actions/using-jobs/setting-default-values-for-jobs
+defaults:
+  run:
+    shell: bash -Eueo pipefail -x {0}
+
+jobs:
+  diagnostics:
+    name: Run diagnostics
+    # This job does not need any permissions
+    permissions: {}
+    runs-on: ubuntu-latest
+    steps:
+      # Note that a duplicate of this step must be added at the top of
```

Comment:
Just a reminder to either align with https://github.com/cisagov/skeleton-generic/pull/201 before merge or to merge this pull request first and update this workflow in the linked pull request before merging it instead.

Reply:
Please see commit 014670f.